From 2547331871679f48054cd712131b519792d7cef6 Mon Sep 17 00:00:00 2001
From: Jason Xing <kerneljasonxing@gmail.com>
Date: Thu, 10 Sep 2020 09:02:18 -0400
Subject: [PATCH 23/27] net-test: use crt namespace when nsperf disables crt.l
 TSO/GSO/GRO

Changing the 'srt' to 'crt' is the right way to disable the features of
the crt.l.

Signed-off-by: Jason Xing <kerneljasonxing@gmail.com>
---
 gtests/net/tcp/bbr/nsperf/nsperf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtests/net/tcp/bbr/nsperf/nsperf.py b/gtests/net/tcp/bbr/nsperf/nsperf.py
index c8b5f697f0a8..ed107c2fa245 100755
--- a/gtests/net/tcp/bbr/nsperf/nsperf.py
+++ b/gtests/net/tcp/bbr/nsperf/nsperf.py
@@ -134,7 +134,7 @@ def setup_veth():
     c += 'ip netns exec srt ethtool -K srt.r tso off gso off gro off\n'
     c += 'ip netns exec mid ethtool -K mid.l tso off gso off gro off\n'
     c += 'ip netns exec mid ethtool -K mid.r tso off gso off gro off\n'
-    c += 'ip netns exec srt ethtool -K crt.l tso off gso off gro off\n'
+    c += 'ip netns exec crt ethtool -K crt.l tso off gso off gro off\n'
 
     # server
     c += 'ip netns exec srv ip addr add 192.168.0.1/24 dev srv.r\n'
-- 
2.34.1

